src/main/kotlin/io/github/projectmapk/jackson/module/kogera/KotlinModule.kt

```diff
@@ -20,26 +20,30 @@ import io.github.projectmapk.jackson.module.kogera.ser.serializers.KotlinSeriali
 import java.util.*
 
 /**
- * @param reflectionCacheSize     Default: 512.  Size, in items, of the caches used for mapping objects.
- * @param nullToEmptyCollection   Default: false.  Whether to deserialize null values for collection properties as
- *                                      empty collections.
- * @param nullToEmptyMap          Default: false.  Whether to deserialize null values for a map property to an empty
- *                                      map object.
- * @param nullIsSameAsDefault     Default false.  Whether to treat null values as absent when deserializing, thereby
- *                                      using the default value provided in Kotlin.
- * @param singletonSupport        Default: DISABLED.  Mode for singleton handling.
- *                                      See {@link io.github.projectmapk.jackson.module.kogera.SingletonSupport label}
- * @param strictNullChecks        Default: false.  Whether to check deserialized collections.  With this disabled,
- *                                      the default, collections which are typed to disallow null members
- *                                      (e.g. List<String>) may contain null values after deserialization.  Enabling it
- *                                      protects against this but has significant performance impact.
- * @param useJavaDurationConversion Default: false.  Whether to use [java.time.Duration] as a bridge for [kotlin.time.Duration].
- *                                      This allows use Kotlin Duration type with [com.fasterxml.jackson.datatype.jsr310.JavaTimeModule].
+ * @param initialCacheSize
+ *   Default: [Builder.DEFAULT_CACHE_SIZE]. See [Builder.withInitialCacheSize] for details.
+ * @param maxCacheSize
+ *   Default: [Builder.DEFAULT_CACHE_SIZE]. See [Builder.withMaxCacheSize] for details.
+ * @param nullToEmptyCollection
+ *   Default: false. See [KotlinFeature.NullToEmptyCollection] for details.
+ * @param nullToEmptyMap
+ *   Default: false. See [KotlinFeature.NullToEmptyMap] for details.
+ * @param nullIsSameAsDefault
+ *   Default: false. See [KotlinFeature.NullIsSameAsDefault] for details.
+ * @param singletonSupport
+ *   Default: false. See [KotlinFeature.SingletonSupport] for details.
+ * @param strictNullChecks
+ *   Default: false. See [KotlinFeature.StrictNullChecks] for details.
+ * @param copySyntheticConstructorParameterAnnotations
+ *   Default false.  See [KotlinFeature.CopySyntheticConstructorParameterAnnotations] for details.
+ * @param useJavaDurationConversion
+ *   Default: false. See [KotlinFeature.UseJavaDurationConversion] for details.
```

Comment:
Previously, duplicate descriptions were provided with `KotlinFeature` and `Builder`, but it was easy to omit updates.
Therefore, only links are now listed.